docs/api/apiv3/components/schemas/week_day_write_model.yml

```diff
@@ -0,0 +1,29 @@
+# Schema: WeekDayWriteModel
+---
+type: object
+description: |-
+  Describes a week day as a working day or a non working day (weekend).
+required:
+  - _type
+  - day
+  - working
+properties:
+  _type:
+    type: string
+    enum:
+      - WeekDay
+  day:
+    type: integer
+    description: |-
+      The week day from 1 to 7. 1 is Monday. 7 is Sunday.
+    readOnly: true
```

Comment:
this is a write model. it should not contain readonly properties.

Reply:
on second thought, the only writable property here is `working`. Hence, this whole write model should be nothing more then:

```
{
  working: true
}
```

Reply:
UPDATE: I saw, that you want to provide both, a single day PATCH and a week PATCH. For the later, of course you need `day` property in here, but still it is not a readonly property.

My question now would be: Why maintaining two endpoints? you could levy the week PATCH to work also for single days. (a collection with only one element.)

Reply:
Not sure about it. I think the endpoints will make sense only once the UI will be developed. If the UI has a save button, then we need to save all at once, so PATCH all. If it updates each time you click on a check box, then it needs to update only one, and I thought it would be "cleaner" to patch the resource directly.

And I did not think about patching with a collection with only a single element too.

For the `readonly`, I probably left it after a copy-paste. Thanks for noticing.

And for the patch, could schema just be:
```
type: array
description: |-
  The array of week days.
minItems: 7
maxItems: 7
items:
  $ref: './week_day_write_model.yml'
```
instead of
```
type: object
required:
  - _type
  - _embedded
properties:
  _type:
    type: string
    enum:
      - Collection
  _embedded:
    type: object
    required:
      - elements
    properties:
      elements:
        type: array
        description: |-
          The array of week days.
        minItems: 7
        maxItems: 7
        items:
          $ref: './week_day_write_model.yml'
```

And as you suggest making it possible to patch only one day, I should set `minItems` to 1.

Reply:
I am indeed a little bit of a fan for batch endpoints :D Especially, because they fit both use cases OOTB, patching a single resource and patching multiple ones. ;) But honestly, I wouldn't mind supporting both, as they seem to have only minor overhead.

Reply:
should be good with 2e2b103